File: Backend/barbershop/barbershop/Models/ResponseDTOs/BranchDTO.cs
namespace barbershop.Models.ResponseDTOs
{
    public class BranchDTO
    {
        public int BranchId { get; set; }
        public string BranchName { get; set; } = null!;
        public string ProvinceCity { get; set; } = null!;
        public string WardCommune { get; set; } = null!;
        public string? LocationDetail { get; set; }
        public bool IsActive { get; set; }
        public string? BranchUrl { get; set; }
        public TimeOnly? TimeOn { get; set; }
        public TimeOnly? TimeOff { get; set; }

        public int Barbers { get; set; } = 0;
    }
}

File: Frontend/barbershop/src/types/ResponseDTOs/branchDTO.ts
export interface BranchDTO {
  branchId: number;
  branchName: string;
  provinceCity?: string;
  wardCommune?: number;
  locationDetail?: string;
  branchUrl?: string;
  timeOn: string;
  timeOff: string;
  isActive?: boolean;
  barbers: number;
  // Thêm các trường khác nếu cần
}